app/components/search/SearchResults/SearchResults.module.scss

```diff
@@ -179,3 +157,60 @@
     display: block;
   }
 }
+
+@media print {
+  // This below styles are for printing out the results on our service/search results page.
+
+  // This sets visibility hidden on all elements except for the searchResultsContainer
+  // and its children to avoid printing irrelevant sections of the page.
+  // This follows the example of this answer https://stackoverflow.com/a/2618980
+  body {
+    visibility: hidden;
+  }
+
+  .searchResultsContainer {
+    visibility: visible;
+  }
+
+  .searchResultsContainer {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    margin: 50px auto;
+    width: 700px;
```

Comment:
Pixels are a bit weird of a unit to use for printing. There is actually an [`in` unit](https://developer.mozilla.org/en-US/docs/Web/CSS/length#in) for inches, which is defined as 96px. We could maybe use that unit here, or maybe we should just use relative units, at least for the horizontal dimension. I have no idea if the `vw` unit works for printing, but that's potentially an option. We might also be able to just use percents.